app/src/main/java/com/example/henzoshimada/feeltrip/FeelTripApplication.java

```diff
@@ -136,6 +134,34 @@ public static UpdateQueueController getUpdateQueueController(){
         return updateQueueController;
     }
 
+    private static Boolean networkAvailable = null;
+    public static void setNetworkAvailable(Boolean b){
+        networkAvailable = b;
+    }
+
+    public static Boolean getNetworkAvailable(){
+        if (networkAvailable == null){
+            networkAvailable = Boolean.FALSE;
+        }
+        return networkAvailable;
+    }
+
+    private static ArrayList<Mood> localMainList = null;
+    private static ArrayList<Mood> localProfileList = null;
+
+    public static ArrayList<Mood> getLocalMainList(){
+        if (localMainList == null){
+            localMainList = new ArrayList<>();
+        }
+        return localMainList;
+    }
+
+    public static ArrayList<Mood> getLocalProfileList(){
+        if (localProfileList == null){
+            localProfileList = new ArrayList<>();
+        }
+        return localProfileList;
+    }
```

Comment:
create 2 global mood lists to store downloaded moods for main page and profile page.